plugins/services/src/js/data/groups/resolvers.ts

```diff
@@ -144,6 +144,27 @@ export function resolvers({ pollingInterval }: ResolverArgs): IResolvers {
             return updateQuota(args.data.id, args.data.quota);
           })
         );
+      },
+      editGroup(
+        _parent = {},
+        args: Record<string, unknown> = {}
+      ): Observable<string> {
+        if (!isGroupCreateArgs(args)) {
```

Comment:
we will likely want different args for `GroupEdit` from `GroupCreate` we could make more fields optional and then make less API calls based on the fields present.